frontend/src/style/bootstrap/attivio-swipe-views.less

```diff
@@ -0,0 +1,20 @@
+.swipe-view-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.swipe-view-navigation-button{
+  font-size: 1.5em;
+  color: gray;
+  cursor: pointer;
+  transition: all .1s ease-in-out;
+}
+
+.swipe-view-navigation-button:hover{
+  color: #2a689c;
+  transform: scale(1.4);
+  background-color: #F1F1F1;
+  border-radius: 10px;
+  padding: .1em;
+}
```

Comment:
Can you add a newline at the end of the file? It seems to be upsetting GitHub...

Reply:
Done